include/GL/glui.h

```diff
@@ -38,8 +38,13 @@
 #include <OpenGL/GL.h>
 #include <OpenGL/GLU.h>
 #else
-#include <GL/gl.h>
-#include <GL/glu.h>
+# ifdef _WIN32
+#  define NOMINMAX
+#  include <Windows.h>
```

Comment:
`#include <windows.h>` I think is more conventional.

Have you seen Windows.h in common usage?

Reply:
Letter case doesn't matter on Windows. I do agree I see lower case ones more often in older c/c++ code. Win10 sdk named the file `Windows.h` though.

The file itself is definitely needed if you are doing any Windows specific stuff. A more likely scenario is that by including some Windows specific header, like `winsock2.h` for networking, `windows.h` would be included for you by those headers. So you don't need to explicitly include it most of the time.